packages/core/src/external/carequality/ihe-gateway-v2/outbound/xca/create/iti38-envelope.ts

```diff
@@ -14,6 +14,7 @@ const findDocumentId = "14d4debf-8f97-4251-9a74-a90016b0af0d";
 const stableDocumentType = "7edca82f-054d-47f2-a032-9b2a5b5186c1";
 const onDemandDocumentType = "34268e47-fdf5-41a6-ba33-82133c465248";
 
+const dateFormat = "YYYYMMDDHHmmss";
```

Comment:
nit: ALL_CAPS for constants

Reply:
Is it a thing?

Reply:
Just checked the Notion Best Practices guide, and this rule isn't there. Yet, I feel like I've been asked to change the names of my constants for places like this many times before

Reply:
For now I am leaving things without all caps for constants. I haven't been asked this before ...